Cut down word count in adv/disadv (#69)
zacaz90/Tutorial10_Group-1: Report.md

Design Thinking Approach
========================
## **Brainstormed Ideas and Reflection**
The following features were concluded from the brainstorming session. 
Feature | Justification
--- | ---

> Enhance Interactive Voice Response unit | Gain initial understanding of customer needs without the need for an in-person RM answering the call. This, in addition to the addition of hold music, were the best ideas ideated to increase customer retention in calls while no RMs were available. We concluded that other ideas such as playing ads while customers were on hold were abandoned as they would lead to negative customer experience.

Reviewer: are we keeping "This, in addition..." derp sentence?

Reviewer 2: Can we make it shorter - In addition, hold music respective of customer demographic was suggested to increase customer retention in calls while no RMs were available. We concluded that other ideas such as playing ads while customers were on hold were abandoned as they would lead to negative customer experience.

Reviewer 2: Or we could get rid of it if it takes up space hahah can't believe we didn't catch it

Reviewer 3: Copied it straight across ahaha good catch, probably just going to remove it - doesn't add a lot

Improve RM questionnaire  | Allows better initial classification of the RM by providing more parameters for CMC to score by.
Added internal wiki of further package information | This was decided to be the best way to have all RMs confident with all packages, so they are more likely to make a sale to customers even if they lack experience with the destination.

> Feedback took implemented  |  Customer feedback tool was added so that the CMC can implement RM score in real-time after the call is undertaken. 

Reviewer: Typos
```suggestion
Feedback tool |  Customer feedback tool was added so that the CMC could implement RM score in real-time after the call is undertaken. 
```

Enhanced RM guidelines  | Natural language processing is embedded in the CMC to enhance the autogenerated RM scripts to enhance sale chances. 

> The biggest issue we, as a group, faced over the course of our project was our unfamiliarity with GitHub. This took out a chunk of each group meeting, as we had to spend time figuring out a correct and consistent workflow. Another issue we found was the lack of face-to-face communication due to COVID-19. This was especially prevalent when filming the video as collating everyone’s individual video and editing became very complicated.

Reviewer: chunk --> portion (more formaliticality?)
```suggestion
The biggest issue we, as a group, faced over the course of our project was our unfamiliarity with GitHub. This took out a portion of each group meeting, as we had to spend time figuring out a correct and consistent workflow. Another issue we found was the lack of face-to-face communication due to COVID-19. This was especially prevalent when filming the video as collating everyone’s individual video and editing became very complicated.
```

Potential Advantages and Disadvantages
======================================
## **Competitive Advantage**

> Developing this new information system, if successful, will lead to a plethora of competitive advantages against other major travel companies looking to increase their customer loyalty or retention. Currently, the major travel company involved is in a great position in the market but has some areas that it is lacking in. For example, the CMC (Call Management Centre) has reduced operating efficiency due to call flow control and call routing issues. This leads to some customers having long wait times and drives up the inbound call cost of the CMC. Through the information system, these areas could be improved and provide many advantages.

Reviewer 2: "Currently, the major travel company involved " for this maybe reword it to help the flow "Current the major travel company is in a position to invest resources into areas they are lacking in."

Reviewer 3: I cut that whole sentence for word count, are you suggesting we bring it back in?

Reviewer 2: Oops I didn't realise it was cut out you can get rid of it

> Lessening the call flow control/routing issues through the development of the new information system will reduce wait times for customers and provide more effective matches with RMs (Relationship Managers). This increase in user experience will improve customer retention, as they see changes being made in the system for their benefit. In addition, implementing an easier method of feedback elicitation could lead to a large uptake in customer loyalty due to their opinions being taken into consideration. The above points will inadvertently put greater pressure on competitors to keep up, providing an extraordinary competitive advantage.

Reviewer 2: Maybe change this "increase" to This "enhancement" in user experience will improve.....

From a more business-centric point of view, the travel company will save costs in handling the increasing customer base. This is because of the information system's ability to reduce per-call handling time, which reduces inbound call costs.

## **Possible Effects of Project Failure**
If the information system is ineffective, there are some considerable adverse effects to first take into account. Ranging from the loss of customers to increased operating costs for no benefit, the adverse effects possible should be weighed into consideration alongside the competitive advantages to most accurately see the possible result of the information system.

By communicating to users that the travel company is implementing a solution to their biggest issues, there is an inherent risk if we fail to deliver. If the solution fails to meet expectation, customers may feel rightly disappointed. If these issues have been ongoing, those who expected an end to their problems may be especially upset and cease interaction with the travel company. There is then a high risk these customers will move to one of the company's direct competitors - a huge competitive disadvantage.

> Additionally, there is a small risk that the increase in tasks performed by CMC may slow its efficiency. This would provide a compounded effect, where the CMC is not only slower but expected to be faster too. This lower efficiency will lead to higher inbound call costs and reduced call flow handling.

Reviewer 2: Additionally, there is a small risk that the increase in tasks performed by CMC may decrease its effectiveness. This would have a run-on effect whereby despite the CMC being newly implemented it will be not only run slower but have less efficiency leading to higher inbound call costs and reduced call flow handling. (Just reworded it a bit)

Reviewer: BTW You can also suggest changes with that ± button. Then they can be committed/changed from the comment

Reviewer 3: The CMC isn't newly implemented, right? I'll still take on the other feedback

Reviewer 3: I'm not entirely sold either. My point in the second sentence was that they expected to improve its effectiveness but actually brought it down so the problem is twice as bad. Which, you know, can be cut I'm not attached to it or anything ahaha
Just double-checking that's the direction you think it should go

Reviewer 2: Yea definitely not newly implemented probably should've said improved. But what you have definitely works its all good

Developing this new information system, if successful, will lead to a plethora of competitive advantages against other major travel companies looking to increase their customer loyalty or retention. Currently, the CMC (Call Management Centre) has reduced operating efficiency due to call flow control and call routing issues. This leads to some customers having long wait times and drives up the inbound call cost of the CMC. Through the information system's implementation, these areas could be improved and provide many advantages.

Lessening the call flow control/routing issues through the development of the new information system will reduce wait times for customers and provide more effective matches with RMs (Relationship Managers). This increase in user experience will improve customer retention, as they see changes being made in the system for their benefit. In addition, implementing an easier method of feedback elicitation could lead to a large uptake in customer loyalty due to their opinions being taken into consideration. The above points will inadvertently put greater pressure on competitors to keep up, providing an extraordinary competitive advantage.

> From a more business-centric point of view, the travel company will save costs in handling the increasing customer base. This is because of the information system's ability to reduce per-call handling time, which reduces inbound call costs.

Reviewer: Minor grammatical suggestion
```suggestion
From a more business-centric point of view, the travel company will save costs in handling the increasing customer base. This is due to the information system's ability to reduce per-call handling time, which reduces inbound call costs.
```

## **Possible Effects of Project Failure**
If the information system is ineffective, there are some considerable adverse effects to first take into account. Ranging from the loss of customers to increased operating costs for no benefit, the adverse effects possible should be weighed into consideration alongside the competitive advantages to most accurately see the possible result of the information system.

> By communicating to users that the travel company is implementing a solution to their biggest issues, there is an inherent risk if we fail to deliver. If the solution fails to meet expectation, customers may feel rightly disappointed. If these issues have been ongoing, those who expected an end to their problems may be especially upset and cease interaction with the travel company. There is then a high risk these customers will move to one of the company's direct competitors - a huge competitive disadvantage.

Reviewer: More shortening (without info loss)
```suggestion
By communicating to users that the travel company is implementing a solution to their biggest issues, there is an inherent risk if we fail to deliver. If the solution fails to meet expectation, customers may feel rightly disappointed particularly if these issues have been ongoing. There is a high risk that they may cease interaction with the travel company and move on to a competitor - a huge competitive disadvantage.
```
